configure.ac

```diff
@@ -1188,7 +1188,11 @@ fi
 AX_BOOST_SYSTEM
 AX_BOOST_FILESYSTEM
 AX_BOOST_THREAD
-AX_BOOST_PROCESS
+
+dnl Opt-in to boost-process
+if test x$with_boost_process != x; then
+  AX_BOOST_PROCESS
+fi
```

Comment:
7588341b8a88370ce3382799d739063965376301
May I suggest to use [`AS_IF`](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/autoconf.html#index-AS_005fIF-1574) macro for [those](https://autotools.io/autoconf/m4sh.html) reasons?